README.md

```diff
@@ -58,31 +58,29 @@ The `KnownTokenId` namespace manages the token IDs of supported cryptocurrencies
 
 Note: Users can transfer between different coins/tokens, not just the same token on different blockchains. For example, it's possible to convert BTC to WBTC when moving from Bitcoin to an EVM network.
 
-
 ### XLink SDK
-The `XLinkSDK` object contains the most important functions of this library, all grouped together. To create it:
+The [`XLinkSDK`](https://docs-typedoc.xlink-sdk.pages.dev/modules/XLinkSDK) object contains the most important functions of this library, all grouped together. To create it:
```

Comment:
the link should be `https://releases-latest.xlink-sdk.pages.dev`; this is the link to the latest version

Reply:
Hello @bolasblack .  Here I think there are two things to get an agreement on:
1. the domain. In my opinion, the link must be relative, that means to point to the same "build/release" the user is actually browing, in order to not confuse him while browsing. On that way, I would make the link:   `(..)[/modules...]` (notice the no domain part)
2. where must it point to? the options are: 
   * point to the whole documentation root document ( the `/` ) which shows the README..
   * point to the actual module definition with its internal artifacts.
   
In this 2 question I think the link actually **IS** in the README and is refers to the module.. letting the user click on the module I assume him wants to navigate to module artifacts.

Because of 1 and 2 I would set it to:

* `[XLinkSDK](/modules/XLinkSDK)`

@bolasblack do you agree?

Reply:
@tenuki totally agreed

Reply:
Sure @bolasblack ! that one we agree too, that's why the emoji - face in the following conversation!